### DISCUSSION TITLE
Welcome to ilovepdf Discussions!

### DISCUSSION THREAD

**Discussion by nabilanavab** [nabilanavab/ilovepdf#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KoshikKumar17** [comment]
Hello

**Comment by l3lack-eyes** [comment]
hello. how to host bot on vps?

**Reply by nabilanavab** [reply]
Currently, there's a database error. Please deploy without the database and use Docker for deployment.
